include/revng/Model/Segment.h

```diff
@@ -79,7 +79,7 @@ class model::Segment : public model::generated::Segment {
 
 public:
   bool contains(MetaAddress Address) const {
-    auto EndAddress = StartAddress() + VirtualSize();
+    auto EndAddress = StartAddress().toGeneric() + VirtualSize();
```

Comment:
Wait segments should already be `Generic`

Reply:
I saw `AArch64` segments failing because of alignment. Hence these changes. I guess PDB importer did something weird? 🤔

Reply:
Do we want to have an assert about segments being generic instead?

Reply:
It should be in `verify`

Reply:
Done: see `Model: make sure segments use generic addresses` for details.